src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.props

```diff
@@ -1,77 +1,77 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <None Condition="Exists('$(MSBuildThisFileDirectory)..\build\native\x86\KernelTraceControl.dll')" Include="$(MSBuildThisFileDirectory)..\build\native\x86\KernelTraceControl.dll">
+    <None Condition="Exists('$(MSBuildThisFileDirectory)..\build\native\x86\KernelTraceControl.dll') And ('$(ProcessorArchitecture)' == 'x86' Or '$(ProcessorArchitecture)' == 'msil')" Include="$(MSBuildThisFileDirectory)..\build\native\x86\KernelTraceControl.dll">
```

Comment:
I was a bit worried because the transformations from RID to `ProcessorArchitecture` happen in `.targets`, but this is in an item, so it's processed later despite being imported earlier. These conditions wouldn't work in a `PropertyGroup` but they're fine here.

Reply:
Thanks.  That's not something that I was aware of.